### PR TITLE
Add kuberlogic logs cli

### PR DESCRIPTION
**Pull request by AvoidMe**

### CODE REVIEW

**Comment by AvoidMe** [comment]
Still working on it

File: modules/dynamic-apiserver/pkg/app/log_list.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+func (h *handlers) LogListHandler(params apiLogs.LogListParams, _ *models.Principal) middleware.Responder {
+	ctx := params.HTTPRequest.Context()
+	pods, err := h.clientset.CoreV1().Pods(kuberlogicNamespace).List(ctx, metav1.ListOptions{})
+	if err != nil {
+		msg := "error listing kuberlogic pods"
+		h.log.Errorw(msg)
+		return apiLogs.NewLogListServiceUnavailable().WithPayload(&models.Error{
+			Message: msg,
+		})
```
Comment (telepenin) [inline modules/dynamic-apiserver/pkg/app/log_list.go:27]: Could you please bump PR to #236 and implement tests in new way?

File: modules/dynamic-apiserver/pkg/app/service_logs_list.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+		})
+	}
+	if len(pods.Items) < 1 {
+		msg := "error listing service pods, no pod is available"
+		h.log.Errorw(msg)
+		return apiService.NewServiceLogsListServiceUnavailable().WithPayload(&models.Error{
+			Message: msg,
+		})
+	}
+	pod := pods.Items[0]
```
[the remark this answers is not in the document]
Comment (AvoidMe) [reply modules/dynamic-apiserver/pkg/app/service_logs_list.go:48]: Done, I filtered pods by labels

File: modules/dynamic-apiserver/pkg/app/service_logs_list.go [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+		if params.ContainerName != nil && container.Name != *params.ContainerName {
+			continue
+		}
+		req := h.clientset.CoreV1().Pods(kls.Status.Namespace).GetLogs(pod.Name, &corev1.PodLogOptions{
+			Container: container.Name,
+		})
+		podLogs, err := req.Stream(ctx)
+		if err != nil {
+			msg := fmt.Sprintf("error listing container '%s' logs", container.Name)
+			h.log.Errorw(msg)
```
[the remark this answers is not in the document]
Comment (AvoidMe) [reply modules/dynamic-apiserver/pkg/app/service_logs_list.go:60]: Done

File: modules/dynamic-apiserver/pkg/app/service_logs_list.go [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+			h.log.Errorw(err.Error())
+			return apiService.NewServiceLogsListServiceUnavailable().WithPayload(&models.Error{
+				Message: msg,
+			})
+		}
+		defer podLogs.Close()
+		buf := new(bytes.Buffer)
+		_, err = io.Copy(buf, podLogs)
+		if err != nil {
+			msg := fmt.Sprintf("error obtaining container '%s' logs", container.Name)
```
[the remark this answers is not in the document]
Comment (AvoidMe) [reply modules/dynamic-apiserver/pkg/app/service_logs_list.go:69]: Done

**Review by telepenin — APPROVED** [review]

**Review by ynnt — APPROVED** [review]

**Pull request merged**